lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -85,6 +90,36 @@ export class PublicClientApplication extends ClientApplication implements IPubli
         if (!account) {
             throw BrowserAuthError.createNoAccountError();
         }
+        const thumbprint: RequestThumbprint = {
+            clientId: this.config.auth.clientId,
+            authority: request.authority || "",
+            scopes: request.scopes,
+            homeAccountIdentifier: account.homeAccountId
+        };
+        const silentRequestKey = ThrottlingUtils.generateThrottlingStorageKey(thumbprint);
+        let response = this.silentRequest.get(silentRequestKey);
+        if (typeof response === "undefined") {
+            response = this.acquireTokenSilentAsync(request, account).then((result) => {
+                this.silentRequest.delete(silentRequestKey);
+                return result;
+            }).catch((error) => {
+                this.silentRequest.delete(silentRequestKey);
+                throw error;
+            });
```

Comment:
```suggestion
            response = this.acquireTokenSilentAsync(request, account)
                .then((result) => {
                    this.silentRequest.delete(silentRequestKey);
                    return result;
                })
                .catch((error) => {
                    this.silentRequest.delete(silentRequestKey);
                    throw error;
                });
```

Fixing up the spacing.